### PR TITLE
ADS-6242 price floors on auction end

### PR DESCRIPTION
**Pull request by hankyates-maven**
Adding price floors to auctionEnd

### CODE REVIEW

**Review by partha0solapurkar — APPROVED** [review]
looks reasonable to me! 

a minor suggestion: since there are currently two places that compute the floor price and both the places should keep calculating it the same way until we remove it from one of the places, it may make sense to abstract the logic out in a function

**Comment by AritraSengupta** [comment]
Yeah I agree with @partha0solapurkar, we should move the rounded floor calculation to its own function. Also please cherry pick this change to stable-next to keep them in sync. Otherwise LGTM

**Comment by vipulnaik** [comment]
Agree with @AritraSengupta with the caveat that I recommend doing merges rather than cherry-picks as merges make for more legible history IMHO.

**Comment by partha0solapurkar** [comment]
oh yeah, I had forgotten about the "merge if posible, otherwise cherry-pick to stable-next" thing. in case it helps, here's the past discussion on that: https://github.com/themaven-net/Prebid.js/pull/59

**Review by partha0solapurkar — APPROVED** [review]

File: modules/mavenDistributionAnalyticsAdapter.js [hunk 1]
```diff
@@ -48,6 +48,14 @@ const MAX_BATCH_SIZE_PER_EVENT_TYPE = 32
  * }} AuctionEventArgs
  */
 
+export const getFloor = (fbr) => {
+  let floor = fbr?.getFloor?.()?.floor ?? null;
```
Comment (bradchoate) [inline modules/mavenDistributionAnalyticsAdapter.js:52]: Personally, I prefer semicolons for Typescript/Javascript, but we should use one or the other style and not both.

**Pull request merged**